### DISCUSSION TITLE
GLPI-Agent 1.8

### DISCUSSION THREAD

**Discussion by g-bougard** [glpi-project/glpi-agent#669] [Announcements]
### Announcement

Blog post: https://glpi-project.org/glpi-agent-release-1-8/

Windows packaging is deeply updated to use most up-to-date libraries.

The [full-inventory-postpone new feature](https://github.com/glpi-project/glpi-agent/discussions/592) is available for all.

There are a lot a fixes and enhancement and I would like to say a great thanks to all the involved people!

As usual, the change log can be read there: https://github.com/glpi-project/glpi-agent/blob/1.8/Changes

**Comment by erique-souza** [comment]
Won't the x86 .msi package for Windows be released? x64 already exists, but compatibility with 32-bit operating systems did not come with it, I believe it is necessary to keep 32-bit Windows systems up to date

**Reply by g-bougard** [reply]
Hi @erique-souza 

I took a lot of time to make the new buildchain working for x64, and x86 became legacy since some time. I don't want to loose more time on that support.

So sorry, 32-bit windows systems will have to be stuck to glpi-agent 1.7.3 release which is still perfect for old platforms which won't evolve in the future.

**Comment by DubanFlp** [comment]
Hi, thanks for the new update today I was testing it but it is not registering me to the server, I do not know how you could help me. 
Version 1.7.3 reports fine.

**Reply by g-bougard** [reply]
Hi @DubanFlp 
check glpi-agent log, but in the case you're using a https url, try latest nightly build which includes a fix for that case.
I will release a new version in a week or 2 to publish that important fix and few other updates.

**Comment by HiopsNerevar** [comment]
Hi! After updating to 1.8, the programs began to split into several records, although the version is the same. That is, if earlier, for example, there were Notepad++ version 8.6.4 - 91 pieces, then version 8.6.7 was divided into 4 entries of 8, 5, 3, 2 pieces. Is this a bug or do I need to configure it somewhere?
Thank you!
![Снимок экрана 2024-05-23 092217](https://github.com/glpi-project/glpi-agent/assets/11775018/4b6102da-d266-4671-ac85-a2b67d525867)

**Reply by g-bougard** [reply]
Hi @HiopsNerevar 

you're speaking about an effect in GLPI but we don't know where this screenshot come from. Please be more accurate in your problem description. As far I know, if this is the software assets panel, it can just mean you have different software versions.

Anyway, this also may be a side-effect of new full-inventory-postpone option we fixed yesterday. So you can try to deploy the [latest nightly build](https://nightly.glpi-project.org/glpi-agent/) and see if that fixes your problem.

**Reply by HiopsNerevar** [reply]
Thanks for the answer. Yes, this is a software panel. As you can see, version 8.6.4 processed normally. This was before the update and version 8.6.7 was after. The versions of Windows and Notepad are identical on all machines, so they shouldn't break up like that. This is not very critical, so I will wait for the 1.9 release and after checking I will write again.

**Reply by HiopsNerevar** [reply]
Hello @g-bougard 
well, in version 1.9 the problem persisted. Even the plugin itself is not indexed correctly. Versions are updated via a vbs script. Computers also have an identical Windows version. There is nothing suspicious in the logs. Any ideas what could help? Or should I describe this as a bug report? 
![image](https://github.com/glpi-project/glpi-agent/assets/11775018/5c1ca6f7-7ccf-4d00-8468-d49efac1e878)

**Reply by g-bougard** [reply]
Hi @HiopsNerevar 

do you have the same problem if you disable `full-inventory-postpone` by setting it to `0` ?

**Reply by HiopsNerevar** [reply]
Hello @g-bougard
I can say in advance that the problem really went away after applying the settings. Now few computers have been updated, but so far there are no duplicates.

**Reply by g-bougard** [reply]
Hi @HiopsNerevar 

first of all, an issue related to full-inventory-postpone was found after 1.8 release. It is fixed in 1.9. But to see an effect in GLPI, I think a software inventory update must at least been detected or a full inventory been forced. So I guess the problem would have been fixed by itself after a while.
